nutridine/src/components/navbar/MenuItem.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Box, Text, useColorModeValue } from "@chakra-ui/react";
+
+export const MenuItem = ({ children, onClick, isActive, ...rest }) => {
+  const activeBg = useColorModeValue("light.primary.500", "dark.primary.600");
+  const paddingSize = 2;
+  const transitionDuration = "0.3s";
+
+  return (
+    <Box
+      as="button"
```

Comment:
This is fine, but there are specific `Button` and `ButtonIcons` that can be used fyi